shoop/xtheme/views/editor.py

```diff
@@ -17,12 +18,17 @@
 from shoop.xtheme.view_config import ViewConfig
 from shoop.xtheme.views.forms import LayoutCellFormGroup
 
+# since layouts will most likely break with multiple cells per row, we are
```

Comment:
I think having `ROW_CELL_LIMIT` as a global constant was a Good Idea. Especially now that the comment here is orphaned.